test-assets/SC1.3.3-star-with-heading.html

```diff
@@ -0,0 +1,12 @@
+<html>
+  <body>
+    <p>*</p>
```

Comment:
```suggestion
    <p>star (*)</p>
```
or
```suggestion
    <p>star</p>
```

Reply:
I disagree. The whole point of these examples is that the ★ (or *) is the only "star" in this code and thus a non-sensory indication is needed (here, the "Examples" heading).